app/models/additional_property_model.rb

```diff
@@ -16,7 +16,8 @@ class AdditionalPropertyModel
   validates :mortgage,
             numericality: { greater_than: 0, allow_nil: true },
             is_a_number: true,
-            presence: { if: :owned_with_mortgage? }
+            presence: true,
+            if: :owned_with_mortgage?
```

Comment:
question - why do we have  `if: :owned_with_mortgage?` on a new line instead of the same line as line 19? i.e.
`presence: true, if: :owned_with_mortgage?`

Reply:
Because the `if` bit doesn't just refer to the `presence` bit any more. It is equally relevant to `is_a_number` and `numericality` now, so having it line up with `presence` but not the other two would be kinda misleading.

That's effectively the bug: the "is_a_number" validation was firing when the property was _not_ owned with a mortgage, because the "if" clause used to be exclusively associated with the "presence" validation

Reply:
Oh...! Thank you.